pkg/pxc/queries/queries.go

```diff
@@ -49,12 +52,36 @@ func New(client client.Client, namespace, secretName, user, host string, port in
 	}, nil
 }
 
+func (p *Database) Status(host, ip string) ([]string, error) {
+	rows, err := p.db.Query("select status from mysql_servers where hostname = ? or hostname = ? limit 1;", host, ip)
```

Comment:
we have `limit 1` why we return an array?

Reply:
Was used for debug, forgot to delete